src/pipecat/services/tencentstt.py

```diff
@@ -0,0 +1,122 @@
+
```

Comment:
we usually name the files `service.py`, so in this case `tencent.py`. Since this would conflict with the submodule `tencent`, I believe we can create a `helpers` submodule. So we would have `services/helpers/tencent/asr/...` and `services/helpers/tencent/common/...`

Reply:
good suggestion